### ISSUE TITLE
iris_grib failure

### ISSUE THREAD

**Issue by tv3141** [SciTools/iris#3070]
Since Iris 2.0 this [documentation example](https://scitools.org.uk/iris/docs/latest/examples/General/polar_stereo.html) fails.

Minimal example:

```python
>>> import iris
>>> iris.load_cube(iris.sample_data_path('polar_stereo.grib2'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/__init__.py", line 373, in load_cube
    cubes = _load_collection(uris, constraints, callback)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/__init__.py", line 313, in _load_collection
    result = iris.cube._CubeFilterCollection.from_cubes(cubes, constraints)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/cube.py", line 143, in from_cubes
    for cube in cubes:
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/__init__.py", line 300, in _generate_cubes
    for cube in iris.io.load_files(part_names, callback, constraints):
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/io/__init__.py", line 210, in load_files
    for cube in handling_format_spec.handler(fnames, callback):
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/fileformats/rules.py", line 429, in load_cubes
    user_callback_wrapper=loadcubes_user_callback_wrapper):
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/fileformats/rules.py", line 350, in _load_pairs_from_fields_and_filenames
    cube, factories, references = _make_cube(field, converter)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/fileformats/rules.py", line 293, in _make_cube
    metadata = converter(field)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris_grib/_load_convert.py", line 2384, in convert
    grib2_convert(field, metadata)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris_grib/_load_convert.py", line 2333, in grib2_convert
    grid_definition_section(field.sections[3], metadata)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris_grib/_load_convert.py", line 1204, in grid_definition_section
    grid_definition_template_20(section, metadata)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris_grib/_load_convert.py", line 835, in grid_definition_template_20
    x_coord, y_coord, scan = _calculate_proj_coords_from_lon_lat(section, cs)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris_grib/_load_convert.py", line 855, in _calculate_proj_coords_from_lon_lat
    x0_m, y0_m = cs.as_cartopy_crs().transform_point(
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/iris/coord_systems.py", line 722, in as_cartopy_crs
    self.true_scale_lat, globe)
  File "/home/user/miniconda/envs/ENV/lib/python2.7/site-packages/cartopy/crs.py", line 1329, in __init__
    raise ValueError('It does not make sense to provide both '
ValueError: It does not make sense to provide both "scale_factor" and "true_scale_latitude". Ignoring "scale_factor".
```

**Comment by tv3141** [comment]
Related to #3065

**Comment by tv3141** [comment]
There is a test for this example. But it is always skipped. https://github.com/SciTools/iris/blob/add8e471d5d6d954b5d2346831a483dae6cb5b1e/docs/iris/example_tests/test_polar_stereo.py#L30

**Comment by tkknight** [comment]
The new Read The Docs style documentation has now been merged to master (not released), see #3752

You can this the gallery entry via: https://scitools-iris.readthedocs.io/en/latest/generated/gallery/general/plot_polar_stereo.html#sphx-glr-generated-gallery-general-plot-polar-stereo-py

I believe  the dependecny on iris-grib has now been removed from the galleries as the example uses a netcdf file now.

Are you happy for this issue to be closed now @tv3141?

**Comment by bjlittle** [comment]
@tkknight LGTM :+1:
